packages/fast-usdc/test/exos/settler.test.ts

```diff
@@ -315,7 +303,7 @@ test('happy path: disburse to LPs; StatusManager removes tx', async t => {
   await eventLoopIteration();
   const { storage } = t.context;
   t.deepEqual(storage.getDeserialized(`fun.txns.${cctpTxEvidence.txHash}`), [
-    { evidence: cctpTxEvidence, status: 'OBSERVED' },
+    { evidence: cctpTxEvidence, status: undefined },
```

Comment:
I'm surprised to see `OBSERVED` replaced by `undefined`.

It makes sense to see the `.observed()` _method_ go away, but not the state.

Reply:
yeah we can't lose that state